doc/src/documentation/architecture/adr/adr004-testing-of-mapping.md

```diff
@@ -0,0 +1,30 @@
+# ADR-004 Reduce mapstructmapper tests to a minimum
+
+## Status
+
+<adr-status status='accepted' />
+
+## Context
+
+Testing is all about showing the functionality of our application. We rely on the fact that the
+frameworks and libraries we use during development are sufficiently tested. Therefore, we can focus on checking our
+logic, when writing tests, especially unit tests.
+
+## Decision
+
+In the tests on the mappers we write tests when we define explicit mapping logic.
```

Comment:
Suggestion: Mappers are only tested if we defined explicit mapping logic.

Reply:
I prefere youre sugesstion. Is changed.